### PR TITLE
fix(abort-controller): add ie11 polyfil

### PR DESCRIPTION
**Pull request by JAdshead**
## Description

Polyfil Abort Controller for ie 11 support.

## Motivation and Context

Reports of bugs in ie11 related to AbortController not being available 

## How Has This Been Tested?

Tested for regression locally. Confirmed fixes reported errors in ie11

## Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation (adding or updating documentation)
- [ ] Dependency update

## Checklist:

- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] These changes should be applied to a maintenance branch.
- [ ] This change requires cross browser checks.
- [ ] Performance tests should be ran against the server prior to merging.
- [ ] This change impacts caching for client browsers.
- [ ] This change impacts HTTP headers.
- [ ] This change adds additional environment variable requirements for One App users.
- [ ] I have added the Apache 2.0 license header to any new files created.

### CODE REVIEW

File: src/client/client.js [hunk 1]
```diff
@@ -18,6 +18,9 @@
  * Main App
  * Set up the global store and kick off rendering
  */
+
+// polyfill for ie 11
+import 'abort-controller/polyfill';
```
Comment (anescobar1991) [inline src/client/client.js:23]: does this automatically only get included in the legacy bundle?

**Comment by anescobar1991** [comment]
Thanks @JAdshead!

File: package.json [hunk 2]
```diff
@@ -83,6 +83,7 @@
     "@americanexpress/one-app-ducks": "^4.1.1",
     "@americanexpress/one-app-router": "^1.1.0",
     "@americanexpress/vitruvius": "^2.0.0",
+    "abort-controller": "^3.0.0",
```
Comment (anescobar1991) [inline package.json:86]: no package-lock.json change?

Comment (JAdshead) [reply package.json:86]: its already provided by fetch-enhancers and was deduped so no change

**Review by infoxicator — APPROVED** [review]

File: src/client/client.js (hunk 1 above)
Comment (JamesSingleton) [reply src/client/client.js:23]: @anescobar1991 yeah it only works on branches from the repo itself, not on forks.

**Comment by JAdshead** [comment]
declining in favour of - https://github.com/americanexpress/one-app/pull/289

**Pull request closed without merging**